src/components/Form/helpers/FormSubmissionHelper.js

```diff
@@ -0,0 +1,50 @@
+import { uploadDocument, updateDocument, runNotification } from '../../../utils';
+
+const makeHandleFormSubmission =
```

Comment:
Add some basic JSDoc about the function is about and add basic types to the parameters

Reply:
fixed